examples/2d/elixir_hyp_diff_harmonic_nonperiodic.jl

```diff
@@ -0,0 +1,59 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the hyperbolic diffusion equations
+
+resid_tol = 5.0e-12 # TODO: Taal, move this parameter to the callback
+equations = HyperbolicDiffusionEquations2D(resid_tol)
+
+initial_conditions = Trixi.initial_conditions_harmonic_nonperiodic
```

Comment:
I noticed that sometimes we use, e.g., `initial_conditions = Trixi.intial_conditions_harmonic_nonperiodic` whereas in my elixir conversions I simply used, e.g., `initial_conditions = initial_conditions_rotor`. Is this a matter of convention or should I edit my elixir files to do it like this?

Reply:
What you did is perfectly fine. We need to decide which boundary conditions to keep in Trixi, see #207. If you have some time, it would be nice if you can note your thoughts on this there.

Reply:
That's marked as a blocking issue before we merge Taal (#200) into `master`.